doc/cascadia/profiles.schema.json

```diff
@@ -134,7 +134,7 @@
             "singleLine": {
               "type": "boolean",
               "default": false,
-              "description": "If true, newlines are removed and whitespace is maintained. If false, whitespace is removed and newlines are maintained."
+              "description": "If true, the copied content will be copied as a single line. If false, newlines persist from the selected text."
```

Comment:
```suggestion
              "description": "If true, the copied content will be copied as a single line (even if there are hard line breaks present in the text.) If false, newlines persist from the selected text."
```

Reply:
Personally, not so sure about this one. A wise man once told me "This rewording adds technical correctness but reduces clarity." haha

Reply:
As a person who _still_ thinks "singleLine" is ambiguous, i disagree: this adds clarity. There are hard like breaks in terminal output. This deletes them.